action/protocol/staking/candidate_statereader.go

```diff
@@ -120,12 +120,12 @@ func (c *candSR) ActiveBucketsCount() uint64 {
 }
 
 // GetStakingStateReader returns a candidate state reader that reflects the base view
-func GetStakingStateReader(sr protocol.StateReader) (CandidateStateReader, error) {
+func GetStakingStateReader(sr protocol.StateReader, candCenterHasAlias bool) (CandidateStateReader, error) {
```

Comment:
The arguments should be kept as the same `func GetStakingStateReader(sr protocol.StateReader) (CandidateStateReader, error) {}`. B/c `commit()` is not used in the reader

Reply:
CandidateStateReader has nothing to do with the hard fork. This hard fork related parameter is only used in CandidateStateManager.

Reply:
resolved by PR3694